dubbo-common/src/main/java/org/apache/dubbo/common/json/impl/AbstractJSONImpl.java

```diff
@@ -0,0 +1,222 @@
+package org.apache.dubbo.common.json.impl;
```

Comment:
Please add ASF license header for this file

Reply:
ok